content/content/Examples/stateless-app-mirror.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Stateless Application Mirror"
+date: 2021-12-16T15:47:14-05:00
+draft: false
+---
+
+This tutorial demonstrates how to mirror a simple stateless
+[PHP Guestbook application](https://kubernetes.io/docs/tutorials/stateless-application/guestbook/)
+using Crane.
+
+# Roadmap
+
+* Deploy Guestbook application in "source" cluster.
+* Extract resources from "source" cluster using Crane Export.
+* Generate transformations to prepare manifests for "destination" cluster using Crane
+    Transform.
+* Apply transformations using Crane Apply.
+* Apply application manifests into "destination" cluster.
+
+# Before You Begin
+
+You will need a "source" and "destination" Kubernetes cluster. You can create an
+environment in minikube with:
+
+```bash
+bash <(curl -s "https://raw.githubusercontent.com/konveyor/crane/main/hack/minikube-clusters-start.sh")
+```
+
+or kind with:
+
+```bash
+bash <(curl -s "https://raw.githubusercontent.com/konveyor/crane-runner/main/hack/kind-up.sh")
+```
+
+You will also need to install `crane`. See the [Installation Guide]({{<ref "/content/Getting Started/installation" >}}).
```

Comment:
![image](https://user-images.githubusercontent.com/8192650/149819951-be68cfad-b206-4ac7-a7ef-c994fd2019f7.png)
the link to Installation guide is not working

Reply:
Are you browsing the preview https://deploy-preview-9--crane-documentation.netlify.app/content/examples/stateless-app-mirror/ ?

Looks okay to me 
![1642443330](https://user-images.githubusercontent.com/569754/149820414-c0a70ed3-a46e-483c-beba-fbf3efbd0ddd.png)

Reply:
no, I was browsing the readme directly on GitHub. Works fine on the preview link